docs/reference/ilm/ilm-tutorial.asciidoc

```diff
@@ -51,18 +48,22 @@ A lifecycle policy specifies the phases in the index lifecycle
 and the actions to perform in each phase. A lifecycle can have up to four phases:
 `hot`, `warm`, `cold`, and `delete`. 
 
-You can define and manage policies through {kib} Management or with the 
-<<ilm-put-lifecycle, put policy>> API.
-
 For example, you might define a `timeseries_policy` that has two phases:
  
 * A `hot` phase that defines a rollover action to specify that an index rolls over when it 
 reaches either a `max_size` of 50 gigabytes or a `max_age` of 30 days.
 * A `delete` phase that sets `min_age` to remove the index 90 days after rollover.
 Note that this value is relative to the rollover time, not the index creation time. 
 
-The underlying put policy request looks like this:
+You can create the policy through {kib} Management or with the 
+<<ilm-put-lifecycle, put policy>> API. 
+To create the policy from {kib}, go to **Management > Elasticsearch > Index Lifecycle Policies**.
+
+image:images/ilm/create-policy.png[]
 
+[%collapsible]
+.API example
```

Comment:
Nit: If a section has label, I try to put it first. I figure this is a better indicator to someone browsing the file of what's next. But I've done it both ways before. Not a big deal either way.
```suggestion
.API example
[%collapsible]
```

Reply:
Applied throughout.